docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -77,8 +77,8 @@ You can print out the Kubernetes manifest for the pod that would be created at r
 Argument precedence
 ^^^^^^^^^^^^^^^^^^^
 
-When building the pod object, there may be overlap between KPO params, pod spec, template and airflow connection.
-In general, the order of precedence is KPO argument > full pod spec > pod template file > airflow connection.
+When KPO defines the pod object, there may be overlap between the :class:`~airflow.providers.cncf.kubernetes.operators.pod.KubernetesPodOperator` arguments.
+In general, the order of precedence is ``pod_template_file``, ``pod_template_dict``, ``full_pod_spec``, and followed by ``V1Pod``, by default.
```

Comment:
the order of precedence is KPO argument > full_pod_spec > pod_template_file > pod_template_dict > V1Pod

Reply:
Thank for the correction. Looking more carefully at
https://github.com/apache/airflow/blob/ba053f79edab54366208aad9d88877a95b789eca/airflow/providers/cncf/kubernetes/operators/pod.py#L900-L914 
https://github.com/apache/airflow/blob/ba053f79edab54366208aad9d88877a95b789eca/airflow/providers/cncf/kubernetes/operators/pod.py#L916-L919
https://github.com/apache/airflow/blob/ba053f79edab54366208aad9d88877a95b789eca/airflow/providers/cncf/kubernetes/operators/pod.py#L963
and `PodGenerator.reconcile_pods()`, I agree that's correct.

However, the wording is still quite confusing. KPO arguments is very vague since both `secrets`, `cmds`, `arguments` (what you meant) and `full_pod_spec` are both KPO arguments.

Do you agree with this suggestion?
```suggestion
In general, the order of precedence is KPO field-specific arguments (e.g., `secrets`, `cmds`, `affinity`), more general templates``full_pod_spec``, ``pod_template_file``, ``pod_template_dict``,  and followed by ``V1Pod``, by default.
```